### PR TITLE
Refactor and Reorganize `.csproj` Configuration

### PR DESCRIPTION
**Pull request by AALUND13**
I've added `CommonSettings.props` to make it easier to change properties. Each `.csproj` file imports the properties from `CommonSettings.props`, so you only have to update one `CommonSettings.props` file instead of changing each of the five `.csproj` files. 

I've implemented this to make life easier for me and for anyone else who wants to contribute to the `Unbound` repository.

## Summary by CodeRabbit

- **New Features**
	- Introduced centralized project configuration through `CommonSettings.props`, enhancing project organization.
	- Enhanced debugging capabilities with new `DebugType` settings: `full` and `embedded`.

- **Bug Fixes**
	- Improved dependency management by updating paths for `0Harmony.dll` and `BepInEx.dll` to relative references.

- **Refactor**
	- Streamlined project files by removing unnecessary properties and simplifying configurations across multiple project files.

### CODE REVIEW

File: CommonSettings.props [hunk 1]
```diff
@@ -0,0 +1,16 @@
+<Project>
+  <PropertyGroup>
+	<RoundsFolder>C:\Program Files (x86)\Steam\steamapps\common\ROUNDS</RoundsFolder>
+	<ManagedFolder>$(RoundsFolder)\ROUNDS_Data\Managed</ManagedFolder>
+	<BepInFolder>../Assemblies</BepInFolder>
```
Comment (Scyye) [inline CommonSettings.props]: Is there a reason for this to be a variable?
If for whatever reason it is, maybe rename it to "BepInExCoreFolder" or something of the sort.
But realistically i dont see the use in this being changed ever.

File: UnboundLib/UnboundCore.cs [hunk 2]
```diff
@@ -80,6 +80,14 @@ private void Awake() {
...
 
             TranslationStrings.INIT();
+
+            // This block of code prevents the game from crashing when quitting
+            Application.quitting += () => {
+                GameObject[] objects = GameObject.FindObjectsOfType<GameObject>();
+                foreach(GameObject obj in objects) {
+                    GameObject.DestroyImmediate(obj);
+                }
+            };
```
Comment (Scyye) [inline UnboundLib/UnboundCore.cs]: Out of scope, undo and open another PR with this change.

**Review by Scyye — CHANGES_REQUESTED** [review]

**Comment by AALUND13** [comment]
Both issues have been solved.

**Comment by Scyye** [comment]
okay lets see if this deletes your thing lol 1 sec

**Comment by Scyye** [comment]
@coderabbitai summary

**Pull request merged**